images/Dockerfile.ubuntu

```diff
@@ -38,14 +40,15 @@ RUN apt install -y \
     squashfs-tools \
     sudo \
     systemd \
-    && apt-get clean
+    systemd-timesyncd \
+    xz-utils \
+  && apt-get clean \
```

Comment:
the clean is not needed anymore: https://github.com/moby/moby/blob/03e2923e42446dbb830c654d0eec323a0b4ef02a/contrib/mkimage/debootstrap#L82-L105

Reply:
Here and elsewhere: it also existed in two places.  The linter seems happy with it, so I've removed it.